### PR TITLE
Ramazan Ince HW_W2 It works but results are not presentable

### PR DESCRIPTION
**Pull request by RamazanInce**

### CODE REVIEW

File: Week2/index.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        FROM country
+        LEFT JOIN city
+        ON country.capital = city.id
+        WHERE country.name = ?`,
+        prompts: ['country'],
+      },
+      {
+        question: `\nPlease enter the name of the region as input1
+to list all spoken languages and leave input 2 blank\n`,
+        sql: `SELECT countryLanguage.language
```
Comment (harditsingh) [inline Week2/index.js]: The languages printed out are not unique, can you please address this?

Comment (harditsingh) [inline Week2/index.js]: The languages printed out are not unique, can you please address this?

Comment (RamazanInce) [reply Week2/index.js]: Ok, I will do so.

**Comment by wouterkleijn** [comment]
Thanks @harditsingh Looking forward to the changes @RamazanInce

**Comment by wouterkleijn** [comment]
@RamazanInce Can you please make the necessary changes?

**Pull request closed without merging**